### PR TITLE
[regression-test](framework) support trino connector tests in multi be and fe env

### PR DESCRIPTION
**Pull request by shuke987**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by shuke987** [comment]
run buildall

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy [hunk 1]
```diff
@@ -89,6 +89,7 @@ class Suite implements GroovyInterceptable {
     final List<Closure> finishCallbacks = new Vector<>()
     final List<Throwable> lazyCheckExceptions = new Vector<>()
     final List<Future> lazyCheckFutures = new Vector<>()
+    final isTrinoConnectorDownloaded = false
```
Comment (924060929) [inline regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy:92]: final field?

**Comment by shuke987** [comment]
run buildall

**Comment by shuke987** [comment]
run buildall

**Comment by shuke987** [comment]
run buildall

**Comment by shuke987** [comment]
run buildall

**Comment by shuke987** [comment]
run buildall

File: be/src/vec/exec/vempty_set_node.cpp [hunk 2]
```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
+// t
```
Comment (BePPPower) [inline be/src/vec/exec/vempty_set_node.cpp:17]: Mistaken operation？

**Comment by shuke987** [comment]
run buildall

File: be/src/vec/exec/vempty_set_node.cpp (hunk 2 above)
Comment (shuke987) [reply be/src/vec/exec/vempty_set_node.cpp:17]: will remove it. In pr pipeline, if be code is not modified, then pre-compiled binary will be used. but yesterday there was a bug in thirdparty.

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy (hunk 1 above)
Comment (shuke987) [reply regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/Suite.groovy]: typo, already fixed

**Comment by shuke987** [comment]
run buildall

**Comment by shuke987** [comment]
run buildall

**Pull request closed without merging**